scripts/codex/guard.ps1

```
@@ -0,0 +1,179 @@
+
+Set-StrictMode -Version Latest
+$ErrorActionPreference = "Stop"
+
+$scriptDir = Split-Path -Parent $MyInvocation.MyCommand.Path
+$repoRoot = (Resolve-Path (Join-Path $scriptDir "..\\..")).Path
+$stateDir = Join-Path $repoRoot ".codex\\state"
+$reviewFlag = Join-Path $stateDir "review-passed"
+$protectedBranches = @("develop", "main", "master")
+$featureTaskPattern = "^feature/TASK-[A-Za-z0-9._-]+$"
+
+function Ensure-StateDir {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Ensure-StateDir' uses an unapproved verb.

(PSUseApprovedVerbs)